doc/installation_linux_manual.md

```diff
@@ -0,0 +1,169 @@
+    <br/>
+    ```sh
+    sudo make uninstall && make clean && make && sudo make -j$(($(nproc)-1)) install
+    ```
+    <br/>
+
+    **FYI:** The shared object will be installed in `/usr/local/lib`, header files in `/usr/local/include`. <br/>
+    The binary demos, tutorials and test files will be copied into `/usr/local/bin` <br/><br/>
+
+    **Note:** Linux build configuration is presently configured to use the V4L2 backend by default. <br/><br/>
+    **Note:** If you encounter the following error during compilation `gcc: internal compiler error` <br/>
+    it might indicate that you do not have enough memory or swap space on your machine. <br/>
+    Try closing memory consuming applications, and if you are running inside a VM, increase available RAM to at least 2 GB. <br/><br/>
+    **Note:** You can find more information about the available configuration options on [this wiki page](https://github.com/IntelRealSense/librealsense/wiki/Build-Configuration).
+
+## Troubleshooting Installation and Patch-related Issues
+
+| Error                                                                                                      | Cause                                                                      | Correction Steps                                                                                                                    |
+|------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------|
+| `git.launchpad... access timeout`                                                                          | Behind Firewall                                                            | Configure Proxy Server                                                                                                              |
+| `dmesg:... uvcvideo: module verification failed: signature and/or required key missing - tainting kernel`  | A standard warning issued since Kernel 4.4-30+                             | Notification only - does not affect module's functionality                                                                          |
+| `sudo modprobe uvcvideo` produces `dmesg: uvc kernel module is not loaded`                                 | The patched module kernel version is incompatible with the resident kernel | Verify the actual kernel version with `uname -r`.<br />Revert and proceed from [Make Ubuntu Up-to-date](#install-dependencies) step |
+| Execution of `./scripts/patch-video-formats-ubuntu-xenial.sh` fails with `fatal error: openssl/opensslv.h` | Missing Dependency                                                         | Install _openssl_ package                                                                                                           |
```

Comment:
Change Xenial with up to date patches

Reply:
Lets add a line about "Multiple UDEV rules" warning

Happens when user install both Debians & from source

Correction: remove the unneeded installation (manual / Deb)

Reply:
Done All